pkg/diff/diff.go

```diff
@@ -682,20 +682,31 @@ func Normalize(un *unstructured.Unstructured, opts ...Option) {
 
 // NormalizeSecret mutates the supplied object and encodes stringData to data, and converts nils to
 // empty strings. If the object is not a secret, or is an invalid secret, then returns the same object.
-func NormalizeSecret(un *unstructured.Unstructured, opts ...Option) {
+func NormalizeSecret(un *unstructured.Unstructured, opts ...Option) (error) {
```

Comment:
Since you are making a breaking change to the function signature, may as well remove the now unused options 🤷